tests/datasource/fluent/test_config.py

```diff
@@ -402,15 +497,15 @@ def test_general_splitter_errors(
 
 @pytest.fixture
 @functools.lru_cache(maxsize=1)
-def from_dict_gx_config() -> GxConfig:
+def from_complex_dict_gx_config() -> GxConfig:
```

Comment:
Please revert the name change as well

Reply:
@Kilo59 Done -- assuming that you will create any fixtures with appropriate name(s) for testing cleaned out `assets` key (if any are needed, that is).